From c4a0af4e085736e2adf0ef94624ef12816858d19 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 11 Apr 2017 16:41:28 +0800
Subject: [PATCH] arm: dts: rk3288-evb: remove limit usage for vopb

There are many notes - Don't use vopb for HDMI, save it for eDP,
let's remove them for kinds of products.

Change-Id: Id8ef9ec8ac853e7b68527f59a9a8870b5a45d8f2
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb.dtsi | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-evb.dtsi b/arch/arm/boot/dts/rk3288-evb.dtsi
index 948227e452b4..6a64478931f6 100644
--- a/arch/arm/boot/dts/rk3288-evb.dtsi
+++ b/arch/arm/boot/dts/rk3288-evb.dtsi
@@ -240,13 +240,6 @@
 	#size-cells = <0>;
 	#sound-dai-cells = <0>;
 	status = "okay";
-
-	/* Don't use vopl for HDMI */
-	ports {
-		hdmi_in: port {
-			/delete-node/ endpoint@1;
-		};
-	};
 };
 
 &saradc {
@@ -283,11 +276,6 @@
 	status = "okay";
 
 	ports {
-		/* Don't use vopb for eDP */
-		edp_in: port@0 {
-			/delete-node/ endpoint@0;
-		};
-
 		edp_out: port@1 {
 			reg = <1>;
 			#address-cells = <1>;
@@ -427,11 +415,6 @@
 
 &vopb {
 	status = "okay";
-
-	/* Don't use vopb for eDP, save it for HDMI */
-	vopb_out: port {
-		/delete-node/ endpoint@1;
-	};
 };
 
 &vopb_mmu {
@@ -440,11 +423,6 @@
 
 &vopl {
 	status = "okay";
-
-	/* Don't use vopb for HDMI, save it for eDP */
-	vopl_out: port {
-		/delete-node/ endpoint@0;
-	};
 };
 
 &vopl_mmu {
-- 
2.35.3

